### PR TITLE
[sensors] Launch `vectornav` in `bringup.launch.py`

### PR DESCRIPTION
**Pull request by mazam32**
# Description
This PR does the following:
- Changed the bringup launch file so that it launches vectornav launch file implicitly
- Provides a new file in documents folder to fix issues with laptops with access denied issues.

### CODE REVIEW

**Review by davidcalderon03 — CHANGES_REQUESTED** [review]
Please remove the build, install, and log files from this PR. You need to build the repo in rover-colon, not src

**Review by davidcalderon03 — APPROVED** [review]
LGTM!

File: urc_localization/src/ned_to_enu.cpp
Comment (yambati03) [inline urc_localization/src/ned_to_enu.cpp:1]: Is this file supposed to be here? It is empty.

File: urc_platform/src/imu_parser.cpp
Comment (yambati03) [inline urc_platform/src/imu_parser.cpp:1]: I don't think we need this IMU parser node anymore. There should be an option in the config file for the `vectornav` package that will automatically publish a ROS 2 `sensor_msgs/Imu` message.

**Review by yambati03 — CHANGES_REQUESTED** [review]
The changes to `bringup.launch.py` look good, but had a few comments about the other checked in code. Also, we'll likely want a way to quickly zero the IMU (not sure if the `vectornav` already comes with a service to do this). I'd also add some instructions for calibrating the IMU and setting the correct magnetic declination.

**Pull request merged**